src/components/Camera/Video.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+import * as React from 'react'
+import { h } from 'preact'
+
+import type { CameraType } from './CameraTypes'
+import classNames from 'classnames'
+import style from './style.css'
+
+import { CameraPure, CaptureActions } from './index.js'
+
+type VideoState = {
+  recording: boolean
+}
+
+export default class Video extends React.Component<CameraType, VideoState> {
+  webcam = null
+
+  constructor(props: CameraType) {
+    super(props)
+    this.state = { recording: false }
```

Comment:
A little later on the state is defined using the class property syntax. Both are totally valid (I prefer the class variable syntax nowadays, as there's less to write) but it's probably worth making them consistent :)

Reply:
> A little later on the state is defined using the class property syntax.

Sorry didn't understand the phrase and the context confused even more, can you rephrase please?